### PR TITLE
BookingSync Universe API docs

### PR DESCRIPTION
**Pull request by Azdaroth**

### CODE REVIEW

File: content/guides/bookigsync-universe-api-early-beta.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+* __website__rentals_read
+* __website__rentals_write
+
+To manage the custom schema for rentals and websites, you will need first to define the schema on website level, under `rentals-custom-data-schema` attribute. It is supposed to be a simple JSON field (key-value store), with attribute names as keys and types as values. You can use the following types: `string`, `integer` and `big_decimal`.
+
+Here is an example request how to define custom schema for rentals:
+
+~~~
+curl --header 'Authorization: Bearer YOUR_TOKEN' --header 'Content-Type: application/vnd.api+json' --request PUT --data '{ "data": { "id": 1, "type": "websites", "attributes": { "rentals-custom-data-schema": { "pool_size": "string" } } } }' https://website.apps.bookingsync.com/api/admin/v1/websites/1
```
Comment (StoneFrog) [inline content/guides/bookigsync-universe-api-early-beta.md]: do we have it described somewhere what attributes might be set?

Comment (Azdaroth) [reply content/guides/bookigsync-universe-api-early-beta.md]: I don't think so, but should be enough to get started. This was mostly written for one PM and apparently this PR has been enough. Better to have it merged as it is than keep referring to a PR.

**Comment by StoneFrog** [comment]
Please resolve content issues and we are good to go

**Comment by Azdaroth** [comment]
@StoneFrog Done

**Pull request merged**